ui/tests/helpers/openapi-driven-attributes.js

```diff
@@ -0,0 +1,2796 @@
+// The constants within this file represent the expected model attributes as parsed from OpenAPI
+// if changes are made to the OpenAPI spec, that may result in changes that must be reflected
+// here as well as ensured to not cause breaking changes within the UI.
+
```

Comment:
I'm not too familiar with OpenAPI, is this file the attributes returned from the data in the OpenAPI endpoint? Is this [OpenAPI docs](https://developer.hashicorp.com/vault/api-docs/system/internal-specs-openapi#sys-internal-specs-openapi)? 😬

Reply:
No, this is the attribute data object that we expect back from `getProps`, which then drive the model attributes. I tried to make it clear in the test comments as well as this file, but open to suggestions!